src/Raven.Client/Documents/Subscriptions/SubscriptionWorkerOptions.cs

```diff
@@ -61,6 +61,7 @@ internal class SubscriptionRedirectData
             public string CurrentTag;
             public string RedirectedTag;
             public Dictionary<string, string> Reasons;
+            public long? RegisterConnectionTimespan;
```

Comment:
Maybe `RegisterConnectionCurationInTicks`? BTW cannot it be just `TimeSpan`?

Reply:
done, I prefer to use primitive types for

Reply:
You changed the name to `RegisterConnectionDurationInTicks` below but here we still have `RegisterConnectionTimespan`

Reply:
done